### ISSUE TITLE
:file show only [present] programms

### ISSUE THREAD

**Issue by aksr** [vifm/vifm#1031]
I use same configuration on multiple computers (some lean, some not so lean...).

I was wondering if it's possible for only `[present]` programs to be shown in `:file` menu?
Also, not having many programs defined, but not present ATM, somewhat **clutters** the `:file` menu.

**Comment by xaizek** [comment]
> I was wondering if it's possible for only `[present]` programs to be shown in `:file` menu?

Nope, as of now the menu always contains all registered commands.
